definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+# Definition of done
+
+This document clarifies, when an product backlog item is done. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api
+* [ ] API diagrams
+* [ ] [Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)
+* [ ] Readme
+* [ ] ROS wiki
+* [ ] Code (sphinx/doxygen)
+* [ ] Good commit messages ([example](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9
+))
+* [ ] Changelog.md updated
```

Comment:
Does this mean running `catkin_generate_changelog` and doing just some manual cleanup so it feels right? Or is there a specific guideline to follow?

Reply:
Every Pull-Request should document its changes. `catkin_generate_changelog` is meant to run once before the release - we split this up into every individual PR - but you could still use `catkin_generate_changelog` if it helps.